vector/src/main/java/im/vector/app/features/onboarding/ftueauth/FtueAuthCombinedLoginFragment.kt

```diff
@@ -70,6 +74,28 @@ class FtueAuthCombinedLoginFragment :
             viewModel.handle(OnboardingAction.UserNameEnteredAction.Login(views.loginInput.content()))
         }
         views.loginForgotPassword.debouncedClicks { viewModel.handle(OnboardingAction.PostViewEvent(OnboardingViewEvents.OnForgetPasswordClicked)) }
+
+        viewModel.onEach(OnboardingViewState::canLoginWithQrCode) {
+            configureQrCodeLoginButtonVisibility(it)
+        }
+    }
+
+    private fun configureQrCodeLoginButtonVisibility(canLoginWithQrCode: Boolean) {
+        if (canLoginWithQrCode) {
+            views.loginWithQrCode.isVisible = true
```

Comment:
Maybe remove the if/else block, keep only the if block content and write
```
views.loginWithQrCode.isVisible = canLoginWithQrCode
```

Reply:
Done.